src/test/java/org/broadinstitute/hellbender/tools/reference/CompareReferencesIntegrationTest.java

```diff
@@ -118,19 +119,109 @@ public void testCompareReferencesMultipleReferencesStdOut() throws IOException{
+
+        final String[] args = new String[] {"-R", ref1.getAbsolutePath() , "-refcomp", ref2.getAbsolutePath(), "-base-comparison", "FIND_SNPS_ONLY", "-base-comparison-output", output.getAbsolutePath()};
+        runCommandLine(args);
+
+        final File actualOutput = new File(output, "hg19mini.fasta_hg19mini_chr2multiplesnps.fasta_snps.tsv");
+        final File expectedOutput = new File(getToolTestDataDir(), "expected.hg19mini.fasta_hg19mini_chr2multiplesnps.fasta_snps.tsv");
+        IntegrationTestSpec.assertEqualTextFiles(actualOutput, expectedOutput);
+    }
+
+    @Test
+    public void testFindSNPsIUPACBases() throws IOException{
         final File ref1 = new File(getToolTestDataDir() + "hg19mini.fasta");
-        final File ref2 = new File(getToolTestDataDir() + "hg19mini_missingchr3.fasta");
-        final File ref3 = new File(getToolTestDataDir() + "hg19mini_missingchr1.fasta");
+        final File ref2 = new File(getToolTestDataDir() + "hg19mini_chr2iupacsnps.fasta");
+        final File output = IOUtils.createTempDir("tempFindSNPs");
 
-        final String[] args = new String[] {"-R", ref1.getAbsolutePath() , "-refcomp", ref2.getAbsolutePath(), "-refcomp", ref3.getAbsolutePath()};
+        final String[] args = new String[] {"-R", ref1.getAbsolutePath() , "-refcomp", ref2.getAbsolutePath(), "-base-comparison", "FIND_SNPS_ONLY", "-base-comparison-output", output.toPath().toString()};
         runCommandLine(args);
+
+        final File expectedOutput = new File(getToolTestDataDir(), "expected.hg19mini.fasta_hg19mini_chr2iupacsnps.fasta_snps.tsv");
+        final File actualOutput = new File(output, "hg19mini.fasta_hg19mini_chr2iupacsnps.fasta_snps.tsv");
+
+        IntegrationTestSpec.assertEqualTextFiles(actualOutput, expectedOutput);
     }
 
+    // FULL_ALIGNMENT tests:
+    @Test
+    public void testFullAlignmentModeMultipleSNPs() throws IOException{
+        final File ref1 = new File(getToolTestDataDir() + "hg19mini.fasta");
+        final File ref2 = new File(getToolTestDataDir() + "hg19mini_chr2multiplesnps.fasta");
+        final File output = IOUtils.createTempDir("tempFullAlignmentSNPs");
+
+        final String[] args = new String[] {"-R", ref1.getAbsolutePath() , "-refcomp", ref2.getAbsolutePath(), "-base-comparison", "FULL_ALIGNMENT", "-base-comparison-output", output.getAbsolutePath()};
+        runCommandLine(args);
+
+        final File expectedOutput = new File(getToolTestDataDir(), "expected.hg19mini.fasta_hg19mini_chr2multiplesnps.fasta.vcf");
+        final File actualOutput = new File(output, "hg19mini.fasta_hg19mini_chr2multiplesnps.fasta.vcf");
+        IntegrationTestSpec.assertEqualTextFiles(actualOutput, expectedOutput);
+    }
+
+    @Test
+    public void testFullAlignmentModeDeletion() throws IOException{
+        final File ref1 = new File(getToolTestDataDir() + "hg19mini.fasta");
+        final File ref2 = new File(getToolTestDataDir() + "hg19mini_chr1indel.fasta");
```

Comment:
Add a comment here documenting that the order in which these are specified determines whether it's an insertion or deletion